Tests/View/ViewHandlerTest.php

```diff
@@ -85,8 +85,16 @@ public function testRegisterHandleExpectsException()
     /**
      * @dataProvider getStatusCodeDataProvider
      */
-    public function testGetStatusCode($expected, $data, $isSubmitted, $isValid, $isSubmittedCalled, $isValidCalled, $noContentCode)
-    {
+    public function testGetStatusCode(
+        $expected,
+        $data,
+        $isSubmitted,
+        $isValid,
+        $isSubmittedCalled,
+        $isValidCalled,
+        $noContentCode,
+        $actualStatusCode
```

Comment:
can you make this default to ``null`` please to have less changes [here](https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1299/files#diff-7476d64421812cdc549a3911c73a9602L115) and to make this PR easier to review?